src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -143,19 +143,14 @@ private static Template readTemplateFromStream(final InputStream stream) throws
         return template;
     }
 
-    public boolean isTemplateNull()
-    {
-        return template == null;
-    }
-
     /**
      * Checks if the template is null.
      *
-     * @return true if it's not.
+     * @return true if the template is null.
      */
-    public boolean doesExist()
+    public boolean isTemplateMissing()
     {
-        return !isTemplateNull();
+        return template == null;
```

Comment:
That specific method wasn't used. There were two. I deleted one and renamed isTemplateNull to isTemplateMissing, and made sure the logic was the same as it was before. Just simpler methods.